app/controllers/whitehall_media_controller.rb

```diff
@@ -2,10 +2,15 @@ class WhitehallMediaController < MediaController
 
 protected
 
+  class NoRecordsFound < StandardError
```

Comment:
I think it might be clearer if this was named `WhitehallAssetNotFound` and inherited from the rails `ActiveRecord::RecordNotFound` exception.

Reply:
I've renamed the exception.  We're using `Mongoid` in this app instead of `ActiveRecord`.  I don't think there's any benefit to inheriting from a Mongoid exception here.